pandas/core/series.py

```diff
@@ -4432,6 +4434,12 @@ def fillna(
             downcast=downcast,
         )
 
+    @Appender(generic._shared_docs["pop"] % _shared_doc_kwargs)
+    def pop(
+        self, item,
+    ):
+        return super().pop(item=item,)
```

Comment:
```suggestion
        return super().pop(item=item)
```
(I think this is a bug in `black`)

Reply:
Sure changed this.